sdk/go/azure/network/functionAzureTrafficController.go

```diff
@@ -60,6 +60,8 @@ import (
 //	$ pulumi import azure:network/functionAzureTrafficController:FunctionAzureTrafficController example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourceGroup1/providers/Microsoft.NetworkFunction/azureTrafficCollectors/azureTrafficCollector1
 //
 // ```
+//
+// Deprecated: azure.network/functionazuretrafficcontroller.FunctionAzureTrafficController has been deprecated in favor of azure.networkfunction/azuretrafficcollector.AzureTrafficCollector
```

Comment:
Ah looks it interprets the change as a deprecation – I'll reset both schema gen commits and build from scratch